config.json

```diff
@@ -2398,6 +2398,22 @@
           "cond"
         ],
         "difficulty": 9
+      },
+      {
+        "slug": "rational-numbers",
+        "name": "Rational Numbers",
+        "uuid": "e2777a03-d890-453e-af5b-d61f60adbbf5",
+        "prerequisites": [
+          "pattern-matching",
+          "basics",
+          "integers",
+          "floating-point-numbers",
+          "guards"
```

Comment:
I think option 2 is the weakest, because `Float.pow` would be too hard to implement, people would search for it and find either `:math.pow` or the 1.12 functions.

I also think that not all students will have 1.12, that being said, one could understand `basics` as including basic math functions, so I don't really know which option to pick either.

Reply:
The concern when Elixir was a lot smaller was that distro's didn't update their binaries very often.  I think with Elixir's growing popularity distro , I'm personally of a stance not worrying too much about supporting legacy versions. I think options 1/3 are both reasonable. If we are looking at v3 in august, by then 1.12 will have been out for a few months making it much more reasonable to assume new comers would be using it

Reply:
Ok, let's go with option 3. I'm convinced by:

>  If we are looking at v3 in august, by then 1.12 will have been out for a few months

For the record:

> one could understand basics as including basic math functions

The way we defined concepts, `basics` includes using integer literals and basic math operators defined in the `Kernel` module (`+`, `-`, `*`, `/`), but using the `Integer` module is part of the `integers` concept 🙂.